sdk/notificationhubs/notification-hubs/test/public/listRegistrations.spec.ts

```diff
@@ -1,32 +1,33 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+import { describe, it, assert, beforeEach, afterEach } from "vitest";
 import {
   AppleRegistrationDescription,
   createAppleRegistrationDescription,
-} from "@azure/notification-hubs/models";
+} from "../../src/models/index.js";
 import {
   NotificationHubsClientContext,
   createRegistration,
   deleteRegistration,
   listRegistrations,
-} from "@azure/notification-hubs/api";
-import { assert, isNode } from "@azure/test-utils";
+} from "../../src/api/index.js";
 import { Recorder } from "@azure-tools/test-recorder";
 import { createRecordedClientContext } from "./utils/recordedClient.js";
+import { isNode } from "@azure/core-util";
 
 describe("listRegistrations()", () => {
   let recorder: Recorder;
   let context: NotificationHubsClientContext;
   const registrationIds: string[] = [];
   const deviceToken = "00fc13adff785122b4ad28809a3420982341241421348097878e577c991de8f0";
 
-  beforeEach(async function () {
+  beforeEach(async (_ctx) => {
     if (!isNode) {
       return;
     }
 
-    recorder = new Recorder(this.currentTest);
+    recorder = new Recorder();
```

Comment:
if we're not passing the context to the recorder, how is it getting the name of the test?

Reply:
This is broken and won't work until I can pass in the context.  So, basically that's a TODO once test-recorder supports vitest.